src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/ContactInformation.tsx

```diff
@@ -140,6 +147,12 @@ export const ContactInformationFragmentContainer = createFragmentContainer(
         name
         email
         phone
+        phoneNumber {
+          isValid
+          international: display(format: INTERNATIONAL)
+          national: display(format: NATIONAL)
+          regionCode
```

Comment:
do we have `phoneNumber` field on type Me? 🤔 

when I run `yarn relay` on this branch, or open ContactInformation step in my browser, I'm getting the error `Unknown field 'phoneNumber' on type 'Me'.`

Reply:
relay-check on the CI is giving the same error.

Reply:
Since the [PR](https://github.com/artsy/metaphysics/pull/3546) is not merged you need to use local MP.

Reply:
then use `yarn sync-schema-local` to update force schema.